recipes/ocrmypdf/meta.yaml

```
@@ -0,0 +1,73 @@
+{% set name = "ocrmypdf" %}
+{% set version = "11.6.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ocrmypdf-{{ version }}.tar.gz
+  sha256: 0f624456a50be0b0bc8c0b59704d159f637616c093a1cabe8bb383706561bcf7
+
+build:
+  number: 0
+  skip: true   # [py2k or win]
```

This looks like it can be `noarch: python`:

```suggestion
  noarch: python
```